Public/Src/Cache/ContentStore/Distributed/Utilities/GrpcDistributedPathTransformer.cs

```diff
@@ -16,7 +18,7 @@ namespace BuildXL.Cache.ContentStore.Distributed.Utilities
     /// </summary>
     public class GrpcDistributedPathTransformer : IAbsolutePathTransformer
     {
-        private static readonly string _localMachineName = Environment.MachineName;
+        private static readonly string _localIpAddress = Dns.GetHostEntry(Dns.GetHostName()).AddressList.FirstOrDefault(ip => ip.AddressFamily == System.Net.Sockets.AddressFamily.InterNetwork).ToString();

```

Comment:
>FirstOrDefault [](start = 105, length = 14)

If I recall you might need to filter out loopback devices and I don't know about the connections in datacenters but there is a chance that there are multiple NIC's with different bandwidth. Should we sort by connection speed rather than picking the first?

Reply:
Moving back to MachineName

---
In reply to: [274111091](https://github.com/Microsoft/BuildXL/pull/120#discussion_r274111091) [](ancestors = 274111091)